dist/tools/codespell/ignored_words.txt

```diff
@@ -106,7 +106,10 @@ dedup
 circularly
 
 # Chang (common name) => Change
-Chang
-
-# chang (common name) => change
 chang
+
+# filp (variable name for file pointer) => flip
+filp
+
+# Ether (Scapy submodule name and means Ethernet in some parts of the code) => Either
```

Comment:
[It's a class](https://github.com/secdev/scapy/blob/master/scapy/layers/l2.py#L211), not a submodule ;-).

Reply:
Should be fixed (directly squashed)